### ISSUE TITLE
Discussion about Fully Sparse Style

### ISSUE THREAD

**Issue by yukang2017** [BraveGroup/FullySparseFusion#1]
Hi,

Thanks for your great work. I really learned a lot in this paper.
I am writing to enquire that would you please consider to include [VoxelNeXt](https://github.com/dvlab-research/VoxelNeXt) in the Table 1 (for nuScenes) and Table 2 (for Argoverse2) for comparison. 

Because VoxelNeXt is also fully sparse, I think this comparison is reasonable. I am sorry that the all statistics are not fully provided in our paper. In the following, I will list the detailed results of VoxelNeXt on nuScenes and Argoverse2.

Regards,
Yukang Chen

**Comment by yukang2017** [comment]
This is the detailed results on Argoverse2 dataset.
It is trained via this [config](https://github.com/open-mmlab/OpenPCDet/blob/master/tools/cfgs/argo2_models/cbgs_voxel01_voxelnext.yaml).
The pre-trained model is provided in this [link](https://drive.google.com/file/d/1YP2UOz-yO-cWfYQkIqILEu6bodvCBVrR/view?usp=share_link).

The results are
```
                                   AP    ATE    ASE    AOE    CDS
ARTICULATED_BUS                  0.210  0.906  0.165  0.565  0.155
BICYCLE                          0.407  0.357  0.244  1.035  0.305
BICYCLIST                        0.349  0.244  0.259  1.301  0.257
BOLLARD                          0.537  0.152  0.468  0.974  0.384
BOX_TRUCK                        0.397  0.588  0.238  0.245  0.316
BUS                              0.397  0.614  0.199  0.631  0.303
CONSTRUCTION_BARREL              0.645  0.164  0.285  0.902  0.505
CONSTRUCTION_CONE                0.445  0.149  0.433  0.945  0.325
DOG                              0.096  0.297  0.455  0.996  0.067
LARGE_VEHICLE                    0.073  0.685  0.301  0.646  0.052
MESSAGE_BOARD_TRAILER            0.000  2.000  1.000  3.142  0.000
MOBILE_PEDESTRIAN_CROSSING_SIGN  0.348  0.115  0.430  1.102  0.251
MOTORCYCLE                       0.449  0.275  0.240  1.038  0.343
MOTORCYCLIST                     0.460  0.268  0.231  1.360  0.338
PEDESTRIAN                       0.632  0.267  0.256  1.410  0.455
REGULAR_VEHICLE                  0.720  0.435  0.186  0.594  0.577
SCHOOL_BUS                       0.270  0.548  0.170  0.274  0.222
SIGN                             0.156  0.371  0.358  0.578  0.118
STOP_SIGN                        0.401  0.226  0.423  0.687  0.300
STROLLER                         0.111  0.253  0.333  0.411  0.089
TRUCK                            0.186  0.492  0.146  0.249  0.157
TRUCK_CAB                        0.184  0.782  0.275  0.365  0.136
VEHICULAR_TRAILER                0.222  0.792  0.194  1.147  0.151
WHEELCHAIR                       0.000  0.334  0.365  1.937  0.000
WHEELED_DEVICE                   0.174  0.404  0.146  1.096  0.133
WHEELED_RIDER                    0.067  0.261  0.335  2.475  0.039
AVERAGE_METRICS                  0.305  0.461  0.313  1.004  0.230
```

**Comment by yukang2017** [comment]
This is the detailed results on nuScenes dataset.

It is trained via this [config](https://github.com/dvlab-research/VoxelNeXt/blob/master/tools/cfgs/nuscenes_models/cbgs_voxel0075_voxelnext_headkernel3.yaml).
The test submission file is provided in this [link](https://drive.google.com/file/d/1wNVjxyTuCE3F88GT_TZSgBgdmkA61Fsi/view?usp=share_link).

The results on val set (this follows "LargeKernel3D in the Table 1" for training settings):
```
mAP: 0.6350                                                                                                                        
mATE: 0.2908                                                                                                                       
mASE: 0.2507                                                                                                                       
mAOE: 0.3586                                                                                                                       
mAVE: 0.2162                                                                                                                       
mAAE: 0.1881                                                                                                                       
NDS: 0.6871                                                                                                                        
car     0.850   0.180   0.152   0.162   0.229   0.202                                                                              
truck   0.591   0.325   0.179   0.131   0.193   0.221                                                                              
bus     0.711   0.335   0.184   0.100   0.351   0.242                                                                              
trailer 0.406   0.561   0.200   0.532   0.189   0.181                                                                              
construction_vehicle    0.242   0.659   0.424   0.942   0.117   0.309                                                              
pedestrian      0.858   0.146   0.271   0.449   0.194   0.095                                                                      
motorcycle      0.711   0.196   0.236   0.341   0.315   0.243                                                                      
bicycle 0.585   0.165   0.264   0.441   0.140   0.011                                                                              
traffic_cone    0.715   0.139   0.318   nan     nan     nan                                                                        
barrier 0.682   0.201   0.278   0.130   nan     nan                                                                                
```

The result on the test set:
```
mAP: 0.6450
mATE: 0.2680
mASE: 0.2381
mAOE: 0.3770
mAVE: 0.2191
mAAE: 0.1268
NDS: 0.6996

mAP for each category
"car": 0.845698347102307
"truck": 0.5297083806547648
"bus": 0.6465212980496758
"trailer": 0.5577569476845065
"construction_vehicle": 0.2874971928696938
"pedestrian": 0.8584051956845637
"motorcycle": 0.7319274811986691
"bicycle": 0.45700489450910387
"traffic_cone": 0.7895269344024632
"barrier": 0.7460310057084403
```

**Comment by liyingyanUCAS** [comment]
No problem. We did not receive the detailed results of VoxelNeXt when we were writing the paper. Now that we have them, we will update them in the table.

**Comment by yukang2017** [comment]
Thank you again for this consideration!
